subworkflows/local/alignment.nf

```diff
@@ -68,7 +67,7 @@ workflow ALIGNMENT {
 
         SAMTOOLS_VIEW( STAR_ALIGN.out.bam.join(SAMTOOLS_INDEX.out.bai), ch_genome_fasta, [] )
 
-        SALMON_QUANT( FASTP.out.reads, salmon_index, gtf, [], false, 'A')
+        SALMON_QUANT( FASTP.out.reads, salmon_index, ch_gtf.map{ meta, gtf ->  gtf  }, [], false, 'A')
```

Comment:
Looks like we should update salmon_quant at some point. Or hopefully the community will do it soon.

Reply:
Yes, I will try to take care of it, but it will break like EVERYONE's RNAseq workflow XD